### PR TITLE
Jnsll tr

### PR DESCRIPTION
**Pull request by Jnsll**
English translation of the index.

### CODE REVIEW

File: en/index.html [hunk 1]
```diff
@@ -122,16 +122,16 @@
...
                 </div>
                 <div class="col-lg-4">
-                    <p>REGOVAR est un logiciel permettant le traitement de données génétiques, depuis la récupération des fichiers produits par les séquenceurs, quelle qu’en soit la technologie, jusqu’à la génération de rapports illustrés et de comptes-rendus d’analyse en passant par les contrôles de qualité, la détection, l’annotation, le filtrage, la priorisation et la visualisation de variants.</p>
+                    <p>REGOVAR is a software program allowing the processing of genetic data, from the retrieval of files which are procuded by sequencers, regardless of the technology, to the generation of illustrated reports and accounts with the quality control, the screening, the annotation, the filtering, the prioritisation and the visualisation of the variants.</p>
```
Comment (Arkanosis) [inline en/index.html]: procuded → produced

**Comment by Arkanosis** [comment]
:thumbsup:

File: en/index.html (hunk 1 above)
Comment (Jnsll) [reply en/index.html]: Thanks ! Good catch, i've corrected it ;).

**Comment by ikit** [comment]
thanks :)

**Pull request merged**